modules/data_release/ajax/FileUpload.php

```diff
@@ -17,68 +17,100 @@
 $user    = $factory->user();
 $config  = $factory->config();
 
-if ($_POST['action'] == 'upload'
-    && $user->hasPermission("data_release_upload")
-) {
+if (!$user->hasPermission('data_release_upload')) {
+    header("HTTP/1.1 403 Forbidden");
+    exit;
+}
+
+if (!file_exists(__DIR__ . "/../user_uploads/")) {
+    error_log(
+        "ERROR: File upload failed. Default user_uploads"
+        . " directory not found."
+    );
+    header("HTTP/1.1 500 Internal Server Error");
+    exit;
+} elseif (!is_writable(__DIR__ . "/../user_uploads/")) {
+    error_log(
+        "File upload failed. Default user_uploads directory"
+        . " does not appear to be writeable."
+    );
+    header("HTTP/1.1 500 Internal Server Error");
+    exit;
+}
```

Comment:
I think this can be deleted now that #5815 is merged. The default upload path is `modules/data_release/user_uploads` which will be checked below